src/libs/ValidationUtils.js

```diff
@@ -274,6 +274,15 @@ function isValidPassword(password) {
     return password.match(CONST.PASSWORD_COMPLEXITY_REGEX_STRING);
 }
 
+/**
+ * @param {String} code
+ * @returns {Boolean}
+ */
+function isValidTwoFactorCode(code) {
+    const result = code.match(CONST.REGEX.BASE64);
+    return Boolean(result) && result[0].length === code.length; // ensure the whole code is base64
```

Comment:
instead of `Boolean(result)` if you're trying to check that result isn't empty something like `result.size() > 0` is clearer.

Reply:
String.match() actually returns `null` when the result is empty. So I cast it to boolean here to make sure the return value is always a boolean.